src/main/java/com/mlaf/hu/helpers/exceptions/SensorNotFoundException.java

```diff
@@ -0,0 +1,6 @@
+package com.mlaf.hu.helpers.exceptions;
+
+public class SensorNotFoundException extends Throwable {
+    public SensorNotFoundException(String sensorId) {
```

Comment:
Geen string representatie of wat dan ook?

Reply:
Ik heb dit laten staan. Een stacktrace van deze exception geeft je SensorNotFoundException met het sensorId, ik denk niet dat er nog iets toegevoegd moet worden.